### ISSUE TITLE
Eggs 0.3.1 broke with Deno 1.5.0

### ISSUE THREAD

**Issue by SteelAlloy** [nestdotland/eggs#104]
## 🐛 Bug Report

```ts
error: TS2345 [ERROR]: Argument of type 'import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord' is not assignable to parameter of type 'import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord'.
  Property '#args' in type 'LogRecord' refers to a different member that cannot be accessed from within type 'LogRecord'.
      handler.handle(record);
                     ~~~~~~
    at https://x.nest.land/std@0.74.0/log/logger.ts:116:22

TS2416 [ERROR]: Property 'format' in type 'ConsoleHandler' is not assignable to the same property in base type 'BaseHandler'.
  Type '(logRecord: import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord) => string' is not assignable to type '(logRecord: import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord) => string'.
    Types of parameters 'logRecord' and 'logRecord' are incompatible.
      Type 'import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord' is not assignable to type 'import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord'.
        Property '#args' in type 'LogRecord' refers to a different member that cannot be accessed from within type 'LogRecord'.
  format(logRecord: LogRecord): string {
  ~~~~~~
    at file:///C:/Users/maela/code/gh/nestdotland/eggs/src/utilities/log.ts:35:3

TS2416 [ERROR]: Property 'format' in type 'FileHandler' is not assignable to the same property in base type 'BaseHandler'.
  Type '(logRecord: import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord) => string' is not assignable to type '(logRecord: import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord) => string'.
    Types of parameters 'logRecord' and 'logRecord' are incompatible.
      Type 'import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord' is not assignable to type 'import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord'.
  format(logRecord: LogRecord): string {
  ~~~~~~
    at file:///C:/Users/maela/code/gh/nestdotland/eggs/src/utilities/log.ts:77:3

TS2322 [ERROR]: Type 'ConsoleHandler' is not assignable to type 'BaseHandler'.
  Types of property 'format' are incompatible.
    Type '(logRecord: import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord) => string' is not assignable to type '(logRecord: import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord) => string'.
      Types of parameters 'logRecord' and 'logRecord' are incompatible.
        Type 'import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord' is not assignable to type 'import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord'.
      console: new ConsoleHandler(debugEnabled ? "DEBUG" : "INFO"),
      ~~~~~~~
    at file:///C:/Users/maela/code/gh/nestdotland/eggs/src/utilities/log.ts:122:7

TS2322 [ERROR]: Type 'FileHandler' is not assignable to type 'BaseHandler'.
  Types of property 'format' are incompatible.
    Type '(logRecord: import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord) => string' is not assignable to type '(logRecord: import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord) => string'.
      Types of parameters 'logRecord' and 'logRecord' are incompatible.
        Type 'import("https://nghbeaead2s4n5znqogrk627qxhfdfterapmd3orofnyqlfusora.arweave.net/aY4SAIAepcb3LYONFXtfhc5RlmSIHsHt0XFbiCy0k6I/log/logger.ts").LogRecord' is not assignable to type 'import("https://x.nest.land/std@0.74.0/log/logger.ts").LogRecord'.
      file: new FileHandler("DEBUG"),
      ~~~~
    at file:///C:/Users/maela/code/gh/nestdotland/eggs/src/utilities/log.ts:123:7

Found 5 errors.
```

## To Reproduce

Install deno 1.5.0

## Environment

- Eggs version: 0.3.1
- Deno version: 1.5.0
- Operating system and version: Windows 10, Ubuntu 20.04

## Additional context

This bug is particularly tough, help is wanted.

**Comment by maximousblk** [comment]
was a deno bug
fixed in denoland/deno#8159
works on [2020.10.29](https://github.com/maximousblk/deno_nightly/releases/tag/2020.10.29) nightly build

**Comment by SteelAlloy** [comment]
Issue solved with Deno 1.5.1 🚀
